tests/FakeItEasy.Tests/FakeTests.cs

```diff
@@ -1,3 +1,6 @@
+using FakeItEasy.Core;
+using FluentAssertions;

```

Comment:
Probably won't be an issue once the tests get moved to Specs as @thomaslevesque has requested, but our convention is to put usings inside the namespace, like `using System` below.